src/Stripe/Entities/StripeBankAccount.cs

```diff
@@ -26,7 +27,26 @@ public class StripeBankAccount : StripeObject
         [JsonProperty("fingerprint")]
         public string Fingerprint { get; set; }
 
+        [JsonProperty("routing_number")]
+        public string RoutingNumber { get; set; }
+
         [JsonProperty("metadata")]
         public Dictionary<string, string> Metadata { get; set; }
+
+        #region Expandable Customer
```

Comment:
I don't see Customer anywhere in the docs - where did you find this? Or did you confirm it with Stripe?